crates/cover_crypt/Cargo.toml

```
@@ -17,8 +18,8 @@ wasm_bindgen = ["js-sys", "wasm-bindgen"]
 default = []
 
 [dependencies]
-cosmian_cover_crypt = { version = "11.0.1", features = ["serialization"] }
-cosmian_crypto_core = "7.0"
+cosmian_cover_crypt = { version = "12.0.0", features = ["serialization", "test_utils"] }
```

If you need the feature `test_utils`, import it as a dev-dependency.
```suggestion
cosmian_cover_crypt = { version = "12.0.0", features = ["serialization"] }
```

---

indeed